shunting-yard-common/src/main/java/com/hotels/shunting/yard/common/io/jackson/SqsMessage.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (C) 2016-2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.shunting.yard.common.io.jackson;
+
+import java.io.Serializable;
+
+import javax.annotation.concurrent.NotThreadSafe;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@NotThreadSafe
+public class SqsMessage implements Serializable {
+  private static final long serialVersionUID = 1L;
+
+  @JsonProperty("Type")
```

Comment:
Why do we need these annotations here but we don't need them for the events coming from Apiary which are also converted from JSON to Java in a similar way?

Reply:
OK.